external/libelf/elf.c

```diff
@@ -25,9 +25,14 @@
  */
 
 #include <libelf.h>
+/*
+#include "/usr/include/sys/exec_elf.h"
```

Comment:
Oops, that should be removed. It's where I got the `#define EM_AMD64    62` from.